tests/migration/online_migration/yast2_migration.pm

```diff
@@ -226,8 +235,10 @@ sub run {
     }
 
     send_key "alt-n";
-    assert_screen 'yast2-migration-startupgrade', 90;
-    send_key "alt-u";
+    if (!is_leap_migration) {
```

Comment:
I think here we can use multi-tag, anyway, this condition works also, you can decide which way to do it. :)